sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -318,6 +310,83 @@
     "check_feature_group_status(orders_feature_group)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add metadata to a feature\n",
+    "\n",
+    "We can put some searchable metadata to the features of the FeatureGroup by using the `UpdateFeatureMetadata` API. The current support metadata fields are `description` and `parameters`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.feature_store.inputs import FeatureParameter\n",
+    "\n",
+    "customers_feature_group.update_feature_metadata(\n",
+    "    feature_name=\"customer_id\",\n",
+    "    description=\"The ID of a customer. It is also used in orders_feature_group.\",\n",
+    "    parameter_additions=[FeatureParameter(\"idType\", \"primaryKey\")],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To confirm that the feature has been updated with the new metadata, we use `DescribeFeatureMetadata` to display that feature."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.describe_feature_metadata(feature_name=\"customer_id\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Feature metadata fields are searchable. We use `search` API with filters to display the specific feature."
```

Comment:
I would rephrase the second sentence to _describe the use case_ this feature is meant to support more accurately.

'with filters to display the specific feature' should be replaced with 'find features with metadata that matches some search criteria'.